tests/func/test_api.py

```diff
@@ -58,6 +63,7 @@ def test_get_url_requires_dvc(tmp_dir, scm):
 
 @pytest.mark.parametrize("remote_url", all_remote_params, indirect=True)
 def test_open(remote_url, tmp_dir, dvc):
+    ensure_dir(dvc, remote_url)
     run_dvc("remote", "add", "-d", "upstream", remote_url)
```

Comment:
The reason why it doesn't work is because gdrive remote needs additional configuration. See TestGDriveRemoteCLI in test_data_cloud.py